tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/util/Constant.java

```diff
@@ -128,6 +132,8 @@
     public static final String PACKAGE_NAME = "PackageName";
     public static final String VARIABLE_NAME_NEW = "NewVariableName";
     public static final String PATH = "Path";
+    public static final String JMS_EP_VAR_NAME = "JSMEndpoint";
```

Comment:
Shall we refactor this class and explain what each variable does in a javaDoc comment ?